internal/cephfs/volume.go

```diff
@@ -119,9 +119,17 @@ func (vo *volumeOptions) getSubVolumeInfo(ctx context.Context, volID volumeID) (
 	return &subvol, nil
 }
 
+type ResizeStatus int64
```

Comment:
The type is exported outside of the package (**R**esizeStaus), but the constants are not (**r**esizeUnknown). Is that intentional? I guess the type can stay internal to the package too.